docs/navigation.md

```diff
@@ -0,0 +1,128 @@
+# Navigation panel
+
+This guide will walk you through the functions of each button in navigation panel. 
+
+# Navigation Button
+
+The first button in the navigation panel is the Navigation button. It is one of the default active buttons.
+
+Navigation button is divided into four parts. It provides the following information:
+
+- Servers
+- Operations
+- Messages
+- Schemas
+
+## Servers
+
+The Servers section displays the name and information of the server you are using to build your AsyncAPI.
```

Comment:
I suggest to just link or reuse to existing concept definitions as we already have them.

Reply:
I could not find the existing definition link.

Reply:
For example server > https://www.asyncapi.com/docs/concepts/server

Reply:
Adding a link as suggested.